ui/app/templates/clients/client.hbs

```diff
@@ -84,6 +84,22 @@
           attributes=model.attributes.attributesStructured
           class="attributes-table"}}
       </div>
+      <div class="boxed-section-head">
+        Meta
+      </div>
+      <div class="boxed-section-body {{if model.meta.attributesStructured "is-full-bleed"}}">
+        {{#if model.meta.attributesStructured}}
```

Comment:
Moving this `if` statement to wrap the `.boxed-section-body` div would be a little easier to read what's happening with the `.is-full-bleed` class.

Reply:
Yeah. I didn't want to repeat the outer element, but you're right that it may be more readable. I'll try it out.